IPs/RSKIP326.md

```diff
@@ -1,56 +1,95 @@
-# Change btcDestinationAddress format in `release_request_received` event
+# Pegout events improvements
 
 |RSKIP          |326           |
 | :------------ |:-------------|
-|**Title**      |Change btcDestinationAddress format in `release_request_received` event |
+|**Title**      |Pegout events improvements |
 |**Created**    |21-JUNE-22 |
-|**Author**     |KI |
+|**Author**     |KI & JT |
 |**Purpose**    |Usa |
 |**Layer**      |Core |
-|**Complexity** |1 |
+|**Complexity** |2 |
 |**Status**     |Draft |
 
 ## Abstract
 
-As of now, the `btcDestinationAddress` field logged by the `release_request_received` event is a hash160 bytes format which is not user friendly. This RSKIP proposes updating the `btcDestinationAddress` field to the string format which is more user friendly.
+Improve pegout related events for a better UX by emitting meaningful events, updating the `btcDestinationAddress` event signature, creating a new `pegout_confirmed` event and only emitting the `add_signature` event if a signature was actually added.
 
 ## Motivation
 
-These changes will improve the user experience of getting event information from the blockchain in a human readable/friendly format. 
+### release_request_received format
+
+These changes will improve the user experience of getting event information from the blockchain in a human readable/friendly format.
```

Comment:
I feel like there's some context missing here. Before this part, we could include a brief description of how the btcDestinationAddress is logged right now (byte array of the hash160 representation), and how we'll improve this (logging the string representation).

Reply:
I like the way you explained the motivation for `New pegout_confirmed event`